src/styles/stars.scss

```diff
@@ -0,0 +1,16 @@
+.stars {
+  display: flex;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+    background-repeat: no-repeat;
+    background-position: center;
+    background-image: url(/src/images/star.svg);
```

Comment:
Ensure that the path to the `star.svg` image is correct. If the image is not located at `/src/images/star.svg`, update the path accordingly.